e2e/fixtures/ssr-target-bundle/src/entries.tsx

```diff
@@ -0,0 +1,28 @@
+import { lazy } from 'react';
+import { defineEntries } from 'waku/server';
+import { Slot } from 'waku/client';
```

Comment:
error here because you import a package from monorepo.

Solution is add it to `references` on `tsconfig.json`